README.md

```diff
@@ -631,6 +631,23 @@ To manage a CurseForge modpack automatically with upgrade support, pinned or lat
 > ```shell
 > docker run ... -e "CF_API_KEY=\$11\$22\$33aaaaaaaaaaaaaaaaaaaaaaaaaa"
 > ```
+> You can also use an [environemnt file](https://docs.docker.com/compose/environment-variables/set-environment-variables/) to load your confguration values. Using an environment file will remove the need to escape your API Key.
+>
+> ```
+> TYPE=AUTO_CURSEFORGE
+> CF_API_KEY=$11$22$33aaaaaaaaaaaaaaaaaaaaaaaaaa
+> ```
+>
+> Configure docker-compose
+> ```yaml
+> minecraft:
+>   env_file:
+>   - ./minecraft-server.env
```

Comment:
Actually `.env` is a special file that doesn't even need to be called out specifically:

https://docs.docker.com/compose/environment-variables/set-environment-variables/#substitute-with-an-env-file

I'm leaning towards that one being documented with just the api key and showing all the other variables can be in the compose file like normal.

I can be easily convinced otherwise since `docker run` still needs the env file specifically mentioned.

Reply:
that's a good point, i will indicate `.env`.